community.md

```diff
@@ -48,32 +82,9 @@ We run an announcement-only, no-marketing-spam mailing list that we will use to
 <input class="cTextfieldstyle" maxlength="90" value="" id="emailUserOS" name="email" placeholder="I consent to join the email list" title="Email" type="text">
 </form> -->
 
-## Want to report issues? 
-
-Oh no, you hit a bump on the road? Please tell us - we want to fix all bugs!
-
-- Found a bug in the compiler, runtime, standard library, or tooling? Create an issue in the <a href="https://github.com/ballerina-platform/ballerina-lang/issues">ballerina-lang</a> repo.
-- Found a bug in the language specification? Create an issue in the <a href="https://github.com/ballerina-platform/ballerina-spec/issues">ballerina-spec</a> repo.
-- Found a bug on the website? Create an issue in the <a href="https://github.com/ballerina-platform/ballerina-dev-website/issues">ballerina-dev-website</a> repo. 
-- Found a security flaw? Send an email to security@ballerina.io. For details, see the <a href="https://ballerina.io/security/">security policy</a>.
-
-## Want to help us make Ballerina popular?!
-
-We know getting a language established is like getting a new religion going. We really need you, our fans, to help spread the word.
-
-Let's start with starring the <a href="https://github.com/ballerina-platform/ballerina-lang">main GitHub repo</a>. Thank you!
-
-Next, let's share the wealth by publishing your code on <a href="https://central.ballerina.io">Ballerina Central</a>. Hope you will do it now.
-
-Also, follow us on Twitter: <a href="https://twitter.com/ballerinalang">@ballerinalang</a>. Tweet with the "#ballerinalang" hashtag.
-
-Do you want to organize a local meetup or hackathon? WSO2 can help with presentation / training content, logistics, swag, and some funds for munchies. Give us a buzz and we’ll get right on it: <a href="mailto:contact@ballerina.io">contact@ballerina.io</a>.
-
-
-## Want to help implement Ballerina?
-Join the project and write some code with us to make Ballerina better! You can hang out with us on the Slack channel, the mailing list, and of course on GitHub.
+## Contribute to Ballerina
 
-Read the <a href="https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md">Contributor Guide</a> so you are aware of the ground rules as you start. Happy dancing!
+Join us and contribute to the source code to make Ballerina better! To be aware of the ground rules as you start, see the <a href="https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md">Contributor Guide</a>. Happy contributing! 

```

Comment:
Since we are adding a contribution guide page for the b.io along with this PR, shall we link that instead of contributing guide of lang repo?

Reply:
Changed the link to "<a href="https://ballerina.io/contribution-guide">Contribution Guide</a>".